### ISSUE TITLE
Training script for the metric model

### ISSUE THREAD

**Issue by wangkua1** [EPFL-VILAB/ViPer#4]
Hi,

It's me again!  Thanks for the great work.  Is it possible to share the training script for the metric model?  After getting it to work, I'm impressed by how well it works, and is interested the in-context fine-tuning described by the paper.  It'd be great to see how you implemented training.  

Thanks!!
Jackson

**Comment by Mowenyii** [comment]
same question

**Comment by sogandstormesalehi** [comment]
Hi! The training code is now public, and you can find the guide to generate the training data in section 3.3 of our paper.
